editor/plugins/texture_editor_plugin.cpp

```diff
@@ -58,6 +58,25 @@ void TexturePreview::_notification(int p_what) {
 	}
 }
 
+void TexturePreview::_update_metadata_label_text() {
+	Ref<Texture2D> texture = texture_display->get_texture();
+
+	String format;
+	if (Object::cast_to<ImageTexture>(*texture)) {
+		format = Image::get_format_name(Object::cast_to<ImageTexture>(*texture)->get_format());
+	} else if (Object::cast_to<StreamTexture2D>(*texture)) {
+		format = Image::get_format_name(Object::cast_to<StreamTexture2D>(*texture)->get_format());
+	} else {
+		format = texture->get_class();
+	}
+
+	metadata_label->set_text(itos(texture->get_width()) + "x" + itos(texture->get_height()) + " " + format);
+}
+
+void TexturePreview::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("_update_metadata_label_text"), &TexturePreview::_update_metadata_label_text);
+}
```

Comment:
You don't need to bind the callback in 4.0, `callable_mp` doesn't need it.

Reply:
Oh, good to know! Removed it already.
Is binding needed in some situations though? I can still see binds to private methods here and there, I wonder if they're just leftovers or if sometimes they're actually needed.

Reply:
They're still needed for stuff like UndoRedo which can't rely on Callables currently.